src/Compiler/Facilities/LanguageFeatures.fs

```diff
@@ -223,6 +223,7 @@ type LanguageVersion(versionText) =
                 LanguageFeature.LowerSimpleMappingsInComprehensionsToFastLoops, languageVersion90
                 LanguageFeature.ParsedHashDirectiveArgumentNonQuotes, languageVersion90
                 LanguageFeature.EmptyBodiedComputationExpressions, languageVersion90
+                LanguageFeature.UnmanagedConstraintCsharpInterop, languageVersion90
```

Comment:
Move it to languageVersion100, NOT languageVersion90.
Also, move it a few lines below to the block with other languageVersion100 features.